packages/web-components/src/components/va-file-input/va-file-input.scss

```diff
@@ -217,4 +217,20 @@ va-button {
   .thumbnail-error {
     color: $vads-color-error-dark;
   }
+
+  #input-error-message.no-file {
+    padding: 0;
+  }
+
+  .visually-hidden {
+    border: 0;
+    clip: rect(0 0 0 0);
+    height: 1px;
+    margin: -1px;
+    overflow: hidden;
+    padding: 0;
+    position: absolute;
+    width: 1px;
+  }
```

Comment:
Because we can't associate the label and hint text across the shadow dom, instead we're sending in the label and hint text and hiding it. [Hiding code from w3](https://www.w3.org/WAI/tutorials/forms/labels/#note-on-hiding-elements)

Reply:
Could we just use the USWDS `.usa-sr-only` class? The component is already importing that [module](https://github.com/department-of-veterans-affairs/component-library/blob/d5700daf4238fb12541c06d2fc6f90e1aab214a7/packages/web-components/src/components/va-file-input/va-file-input.scss#L5). Or maybe there is a functional difference between these two classes?